### PR TITLE
Config docker to run local more easy

### PR DESCRIPTION
**Pull request by Cristuker**
I add a docker compose file to the project and update the readme to make more easy for the beginners run the database. With this PR they only need to install docker compose and run `docker-compose up -d` no need to install db or search for how to do this.

PS: i leave a example how to build a url for postgres

### CODE REVIEW

**Comment by Cristuker** [comment]
This feature is ready for review

File: README.md [hunk 1]
```diff
@@ -83,19 +84,25 @@ cp .env.example .env
 
 #### 5. Set up database schema
```
Comment (devkiran) [inline README.md:85]: Can you change the heading, now both step 5 and 6 has same heading. I would like to mark the `docker-compose` step as optional if the developers want to spin on a database using Docker or they can use any database they want.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: admin
+      POSTGRES_USER: admin
+      POSTGRES_DB: starter-kit
```
Comment (devkiran) [inline docker-compose.yml:9]: Rename the database name to `saas-starter-kit`

**Review by devkiran — CHANGES_REQUESTED** [review]

**Review by devkiran — APPROVED** [review]

**Comment by devkiran** [comment]
Thank you for your time and this contribution.

**Pull request merged**